dapp/src/components/buySell/SwapHomepage.js

```diff
@@ -1,42 +1,41 @@
-import React, { useState, useEffect, useRef } from 'react'
-import { fbt } from 'fbt-runtime'
-import { useStoreState } from 'pullstate'
-import { ethers, BigNumber } from 'ethers'
-import { get, find } from 'lodash'
-
-import AccountStore from 'stores/AccountStore'
-import TransactionStore from 'stores/TransactionStore'
-import ContractStore from 'stores/ContractStore'
-import ApproveModal from 'components/buySell/ApproveModal'
 import AddOUSDModal from 'components/buySell/AddOUSDModal'
-import ErrorModal from 'components/buySell/ErrorModal'
-import DisclaimerTooltip from 'components/buySell/DisclaimerTooltip'
+import ApproveButtonLogic from 'components/buySell/ApproveButtonLogic'
 import ApproveCurrencyInProgressModal from 'components/buySell/ApproveCurrencyInProgressModal'
+import ApproveModal from 'components/buySell/ApproveModal'
+import ErrorModal from 'components/buySell/ErrorModal'
+import SettingsDropdown from 'components/buySell/SettingsDropdown'
+import SwapCurrencyPill from 'components/buySell/SwapCurrencyPill'
+import PillArrow from 'components/buySell/_PillArrow'
 import { currencies } from 'constants/Contract'
-import { providersNotAutoDetectingOUSD, providerName } from 'utils/web3'
+import { ethers } from 'ethers'
+import { fbt } from 'fbt-runtime'
+import withIsMobile from 'hoc/withIsMobile'
 import withRpcProvider from 'hoc/withRpcProvider'
-import usePriceTolerance from 'hooks/usePriceTolerance'
 import useCurrencySwapper from 'hooks/useCurrencySwapper'
-import BuySellModal from 'components/buySell/BuySellModal'
-import SwapCurrencyPill from 'components/buySell/SwapCurrencyPill'
-import PillArrow from 'components/buySell/_PillArrow'
-import SettingsDropdown from 'components/buySell/SettingsDropdown'
-import { isMobileMetaMask } from 'utils/device'
```

Comment:
Curious do you have some sort of IDE that re-arranged all these imports alphabetically?